src/utils/parsingModel.json

```diff
@@ -2696,5 +2696,201 @@
         }
       ]
     }
+  },
+ "Activité du journal des accès/Activités _ liste des services Google auxquels vos.csv": {
+    "language": "fr",
+    "comment": "list all google services used by the suer",
+    "file_structure_properties": {
+      "header": true,
+      "delimiter": ","
+    },
+    "entries": [
+      {
+        "action_type": "Google services list"
+      },
+      {
+        "data_origin": "Inferred"
+      },
+      {
+        "data_type": "Contractual"
+      },
+      {
+        "platform": "google"
+      },
+      {
+        "timestamp": ""
+      },
+      {
+        "details": ""
+      }
+    ]
+  },
+ "Activité du journal des accès/Appareils _ liste des appareils (par exemple, Nest.csv": {
+    "language": "fr",
+    "comment": "list all devices linked to the account",
+    "file_structure_properties": {
+      "header": true,
+      "delimiter": ","
+    },
+    "entries": [
+      {
+        "action_type": "list of all devices"
+      },
+      {
+        "data_origin": "Observed"
+      },
+      {
+        "data_type": "Technical"
+      },
+      {
+        "platform": "google"
+      },
+      {
+        "timestamp": ""
+      },
+      {
+        "details": ""
+      }
+    ]
+  },
+ "Google Pay/Envois et demandes d_argent": {
```

Comment:
En EN j'ai le path suivant : `Google Pay/Money remittances and requests/Money remittances and requests.csv`

Reply:
tu peux tester idem